packages/frontend/src/pages/SavedExplorer.tsx

```diff
@@ -53,6 +55,7 @@ const SavedExplorer = () => {
 
     return (
         <ExplorerProvider
```

Comment:
Splitting this component, will break the save transition, and trigger extra requests , like In here . So not worth it for now.
[Screencast from 21-03-23 14:53:35.webm](https://user-images.githubusercontent.com/1983672/226861449-7031367e-311b-433d-be36-5815e290af30.webm)